stable/rabbitmq/templates/svc.yaml

```diff
@@ -42,8 +42,8 @@ spec:
     targetPort: stats
 {{- if .Values.metrics.enabled }}
   - name: metrics
-    port: 9090
-    targetPort: metrics
+    port: {{ .Values.metrics.port }}
+    targetPort: 9090
```

Comment:
I'd prefer to keep here the `metrics` targetPort and not to hard code the port number.
Could you make the change ?

Reply:
@rafariossaa  I don't want it this way either, however, the underlying image has a hard-coded expose on 9090 only.  https://github.com/bitnami/bitnami-docker-rabbitmq-exporter/blob/master/0/debian-9/Dockerfile#L15